moveit_planners/ompl/ompl_interface/src/detail/constrained_goal_sampler.cpp

```diff
@@ -54,7 +54,7 @@ ompl_interface::ConstrainedGoalSampler::ConstrainedGoalSampler(
 {
   if (!constraint_sampler_)
     default_sampler_ = si_->allocStateSampler();
-  logDebug("Constructed a ConstrainedGoalSampler instance at address %p", this);
+  ROS_DEBUG_NAMED("constrained_goal_sampler", "Constructed a ConstrainedGoalSampler instance at address %p", this);
```

Comment:
Again, logger name should be prefixed with `ompl`: `ompl.constrained_goal_sampler`.